BUILDING.md

```diff
@@ -260,6 +260,11 @@ PASS
 ok  	github.com/containerd/containerd	4.778s
 ```
 
+> *Note*: in order to run `sudo go` you need to
+> - either keep user PATH environment variable. ex: `sudo "PATH=$PATH" env go test <args>`
+> - or use `go test -exec` ex: `go test -exec sudo -v -run "TestTarWithXattr" ./archive/ -test.root`
+> - or use unsafe and distro specific way: append go-path to secure_path variable in /etc/sudoers by using `sudo visudo` (systemwide change can cause unintended results)
```

Comment:
Can we please remove this last line?  I think the first two suggestions are enough for all users building a container runtime.

Reply:
Got it, commit updated.